docs/admins/guide.rst

```diff
@@ -0,0 +1,173 @@
+
+   .. admonition:: Example
+
+      **Lesson 1: Introduction to Decimals**
+       * Given a decimal with a zero in the one's place, recognize that the decimal is less than one.
+       * Given a decimal number, identify the whole number and fractional parts of the decimal.
+
+      **Lesson 2: Decimal concepts**
+       * Given a decimal number, state the place value of each digit.
+       * Given a decimal with hundredths place value, round it to the nearest tenths.
+   
+   You may choose to repeat similar skills in different lessons.
+
+
+   4. **Prerequisite skills.** Specify prerequisite skills that a learner would need to have that may not be part of the topic scope. The prerequisites should also include specific skills from previous lessons in the series. These will be used to generate random review questions at the start of each lesson to determine whether the student has indeed mastered the skills they need for the present topic. 
+
+Developing a Story Outline
+---------------------------
+
+Usually a topic will have one story that will span across different lessons. At this stage, you will draft a high-level story for each lesson.
+
+Stories are what make Oppia lessons so engaging for learners. You don't have to overdo it with the story details and plot—after all, the concept being taught should be driving the lessons forward. You want just enough of a story arc to keep learners engaged and empathetic with your main character. Like any good story, there should be a challenge for your protagonist (a possible trap or misconception) that they overcome; with the learner also feeling that accomplishment by the end of the lesson.
+
+A good way to approach this step is to think of a practical, real-world setting that best demonstrates the concept being taught.
+
+.. admonition:: Example
+
+   A good real-world example for Decimals is prices in dollars and cents. A story emanating from this idea might be a young girl in a shop grappling with a price as she tries to relate it to decimal concepts. This setting might then span several lessons as the character learns how to read the price, connects the tenths and hundredths place values to dimes and pennies, compares decimal place values and so on.
+
+Your high-level story outline will include:
+
+ * A brief overview of the characters, settings and premise.
+ * One to two paragraphs describing what happens in each lesson.
+
+.. _skilltable:
+
+Creating a Skill Table
+-----------------------
+
+The Skill Table is where you will build upon the previous two steps and further elaborate the skills you have identified. Before you create the table however, list the Prerequisite skills, Acquired skills and Practiced skills for the lesson.
+
+ * **Prerequisite skills**: Skills the learner would need before starting the lesson.
+ * **Acquired skills**: Skills the learner will acquire by the end of the lesson.
+ * **Practiced skills**: Skills that will be taught and practiced during the lesson.
```

Comment:
Is this just for the creator's reference? Since, there is no field where the creator can input 'Practiced Skills' in the editor..

Reply:
This section was based off the [current documented process](https://github.com/oppia/oppia/wiki/Lesson-Creation-Guide#Table) and [this example](https://docs.google.com/document/d/1GNP_slUbT1ZjA9BhDquvSTEWEIBlBgM2NdXcaCDSeVo/edit?usp=sharing), but after speaking with Sean I understand the process is changing and will be moved to the Oppia interface (instead of Google Docs as was done so far).
So this section actually will be updated when the process of creating a skill tree and table etc. migrates to the Oppia interface.

Reply:
Hi @audstav, please feel free to update this now with the intended eventual future process. The "chapter notes" sections in the story fields can be used to collect this information.

Reply:
@aks681 @seanlip Sounds good, I will submit another commit to this PR.